src/components/CurrencyInput/components/SimpleCurrencyInput/SimpleCurrencyInput.js

```diff
@@ -93,7 +94,11 @@ const SimpleCurrencyInput = ({
   numberMask,
   ...props
 }) => (
-  <MaskedInput
+  <TextMaskInput
+    guide={false}
+    render={(ref, { defaultValue, ...renderProps }) => (
+      <Input value={defaultValue} {...renderProps} deepRef={ref} />
+    )}
```

Comment:
Nice component!! 
Didn't knew this one... can be quite useful 🙂 👍

Reply:
Well, it’s gone from Circuit UI now 😆

Reply:
Of course, you can use [`react-text-mask`](https://www.npmjs.com/package/react-text-mask) directly instead.